scripts/globals/abilities/super_jump.lua

```diff
@@ -22,4 +23,18 @@ end;
 -----------------------------------
 
 function onUseAbility(player,target,ability)
-end;
\ No newline at end of file
+
+    -- Reduce 99% of total accumulated enmity
+    if (target:isMob()) then
+        target:lowerEnmity(player, 99);
+    end
+    
+    ability:setMsg(0);
+    
+    -- If the Dragoon's wyvern is out and alive, tell it to use Super Climb
+    local wyvern = player:getPet();
+    if (wyvern ~= nil and player:getPetID() == PET_WYVERN and wyvern:getHP() > 0) then 
+        wyvern:useJobAbility(636, wyvern);
+    end
+
```

Comment:
the only reason I didn't implement super jump when I did the rest of the earlier drg refactors was because I don't know how it behaves exactly (can party members cast on you? can they even target you?)
My first thought was to just exclude them from all targetfinding, but I don't remember if I tried it or not (and I guess it would probably be a custom status effect or some kind of flag on a timer, but it doesn't really matter which of those is chosen)